.github/workflows/eden-builder.yml

```diff
@@ -23,9 +23,7 @@ jobs:
                   REGISTRY="ghcr.io"
                   IMAGE="${REGISTRY}/${{ github.repository_owner }}/eden-builder"
                   TAGS="${IMAGE}:${{ github.sha }}"
-                  if [[ $GITHUB_REF == ref/head/main ]]; then
-                    TAGS="${TAGS},${IMAGE}:latest"
-                  fi
+                  TAGS="${TAGS},${IMAGE}:latest"
```

Comment:
we could calculate a version ENV VAR in the make file based on the hash of the githb repo , like this:

https://github.com/eoscostarica/eosio-dashboard/blob/10640b27a4c64d9603008f0a70aad16df3a22a48/utils/meta.mk#L3